docs/administrator-documentation/moderne-platform/references/reporting.md

```diff
@@ -23,7 +26,11 @@ Audit logs are accessible from `https://<TENANT>.moderne.io/admin/audit-logs`, a
 
 To download audit logs, use the "Export to CEF" button: ![](./assets/audit-log-download.png)
 
-## Other reports (recipe runs and commits)
+## Usages by organization
+
+The usages report is accessible from `https://<TENANT>.moderne.io/devcenter/{organization}`
+
+## Other reports (recipe runs, commits and usages)
```

Comment:
~usages~ -> usage

Confused about this too -- didn't you just mention it above?

Reply:
I'm going to remove the `Usage by organization` section I don't think its necessary